Assets/Script/GameManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System;
+
+public class GameManager : MonoBehaviour
+{
+    public Text PlayTime;
+    private TimeSpan startTime = new TimeSpan(0, 10, 00);
+
```

Comment:
```suggestion
private TimeSpan remainingTime;
```

To count remaining time